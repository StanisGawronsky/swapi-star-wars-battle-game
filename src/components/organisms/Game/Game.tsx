import React from "react";
import { ChampionType, StarshipType } from "~/types/index";
interface IProps {
  props: {
    champions: ChampionType[];
    starships: StarshipType[];
  };
}
const Game: React.FC<IProps> = (props: IProps) => {
  console.log(props);
  return <div></div>;
};

export default Game;
