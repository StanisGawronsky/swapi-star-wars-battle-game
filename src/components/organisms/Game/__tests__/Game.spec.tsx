import React from "react";
import { render } from "@testing-library/react";
import Game from "../Game";

describe("<Game />", () => {
  it("renders correctly", () => {
    const { container } = render(<Game />);
    expect(container).toMatchSnapshot();
  });
});
