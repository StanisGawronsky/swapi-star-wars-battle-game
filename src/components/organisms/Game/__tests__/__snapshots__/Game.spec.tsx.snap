// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Game /> renders correctly 1`] = `
<div>
  <div>
    <div>
      <div>
        <label>
          Choose your player type
        </label>
        <select>
          <option
            value="champions"
          >
            champions
          </option>
          <option
            value="starships"
          >
            starships
          </option>
        </select>
      </div>
      <div>
        <label>
          Choose your attribute
        </label>
        <select>
          <option
            value="height"
          >
            height
          </option>
          <option
            value="mass"
          >
            mass
          </option>
          <option
            value="birth_year"
          >
            birth year
          </option>
        </select>
      </div>
    </div>
    <div>
      <div />
    </div>
    <button>
      play
    </button>
  </div>
</div>
`;
